### PR TITLE
A small readme for the eth_capetown onboarder

### PR DESCRIPTION
**Pull request by agatsoh**
A small readme for the user to get started with the onboarder script.

### CODE REVIEW

File: tools/onboarder/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## The Onboarder Script
+This script will generate an Ethereum keystore file in json format and will load the account with testnet ether as well as respective tokens. Run
+```
+python onboarder.py
```
Comment (czepluch) [inline tools/onboarder/README.md:4]: Does this actually work? I only tried with the binary.

**Comment by czepluch** [comment]
Shouldn't this me in the master branch that we create other branches from?

**Comment by agatsoh** [comment]
@christianbrb wanted it for this one because we have a link from the developer portal to the onboarding script in `eth_capetown_onboarder` branch.

**Comment by agatsoh** [comment]
Here is the link https://developer.raiden.network/#Examples

**Comment by christianbrb** [comment]
It's just a quick fix, which must not take long or need to be very elaborate - everything is better than nothing

**Comment by agatsoh** [comment]
Solves #5

File: tools/onboarder/README.md (hunk 1 above)
Comment (czepluch) [reply tools/onboarder/README.md:4]: @agatsoh ?

Comment (agatsoh) [reply tools/onboarder/README.md:4]: Hello Jacob I tested it, and it works !!!

**Comment by palango** [comment]
Can we change the link instead, I guess this not be updated, but `master` will. For example the API calls are wrong there, but already fixed on `master`.

**Pull request merged**